docs/cli.md

```diff
@@ -223,16 +223,38 @@ These commands will execute two different project revisions based on the given G
 
 ### Pipeline parameters
 
-Pipeline scripts can use an arbitrary number of parameters that can be overridden using the command line or Nextflow configuration files. Any script parameter can be specified on the command line by prefixing the parameter name with double-dash characters. For example:
+Pipeline scripts can define *parameters* which can be overridden on the command line.
+
+Parameters can be declared in the main script:
+
+```nextflow
+params.alpha = 'default script value'
+
+workflow {
+  println "alpha = ${params.alpha}"
+}
+```
+
+Or in a config file:
+
+```groovy
+params {
+  alpha = 'default config value'
+}
+```
+
+The above parameter can be specified on the command line as `--alpha`:
 
 ```console
-$ nextflow run <pipeline> --foo Hello
+$ nextflow run <pipeline> --alpha Hello
 ```
 
-Then, the parameter can be accessed in the pipeline script using the `params.foo` identifier.
+:::{note}
+Parameters that are specified on the command line without a value are set to `true`.
+:::
 
 :::{note}
-When the parameter name is formatted using `camelCase`, a second parameter is created with the same value using `kebab-case`, and vice versa.
+Parameters that are specified on the command line with `kebab-case` names are automatically converted to `camelCase`. For exapmle, a parameter defined as `fooBar` in the pipeline script can be given on the command line as `--fooBar` or `--foo-bar`.
```

Comment:
```suggestion
Parameters specified on the command line in kebab case (e.g., `kebab-case`) are automatically converted to camel case (e.g., `camelCase`). Because of this, a parameter defined as `fooBar` in the pipeline script can be specified on the command line as `--fooBar` or `--foo-bar`.
```

Reply:
Note that the vice versa was dropped here, so at the moment it reads that only kebab case are converted to camel case.